libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -839,15 +840,18 @@ bool AP_AHRS::_airspeed_estimate(float &airspeed_ret) const
 
     switch (active_EKF_type()) {
     case EKFType::DCM:
+        state.airspeed_estimate_status = ASPD_Status::DCM_SYNTHETIC;
         return dcm.airspeed_estimate(get_active_airspeed_index(), airspeed_ret);
 
 #if AP_AHRS_SIM_ENABLED
     case EKFType::SIM:
+        state.airspeed_estimate_status = ASPD_Status::SIM;
         return sim.airspeed_estimate(airspeed_ret);
 #endif
 
 #if HAL_NAVEKF2_AVAILABLE
     case EKFType::TWO:
+        state.airspeed_estimate_status = ASPD_Status::DCM_SYNTHETIC;
```

Comment:
I know this is the same as DCM but will it cause confusion for EK2 users? Might be worth having a EK2 enum with a comment saying that is really DCM. Comments on the same line get parsed into the docs.

Reply:
can do

Reply:
done